code/datums/uplink/badassery.dm

```diff
@@ -71,7 +71,7 @@
 
 /datum/uplink_item/item/badassery/marshallbadge
 	name = "Marshal's Badge"
-	item_cost = 1
+	item_cost = 0
```

Comment:
If we're gonna make it 0 we need to make it buyable only once or else some chucklefuck will shit out tons of them.

Reply:
If you don't know how to do that just leave the price at 1 TC for now

Reply:
fine, will leave it at 1 TC